test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -151,7 +151,8 @@ void spellCheckNetworkNamesInExpectField(json_spirit::mArray const& _expects)
 		json_spirit::mObject const& expectObj = expect.get_obj();
 			ImportTest::parseJsonStrValueIntoVector(expectObj.at("network"), netlist);
 			for (string const& networkName: netlist)
-				(void)stringToNetId(networkName);
+				if (networkName != "ALL") // "ALL" is allowed as a wildcard.
```

Comment:
should we use a c_ALL  string constant for this?

Reply:
Yes.

Reply:
Will be done https://github.com/ethereum/cpp-ethereum/issues/4453